print_docs.py

```diff
@@ -79,6 +81,63 @@
 # root directory of mathlib.
 local_lean_root = os.path.join(root, cl_args.r if cl_args.r else '_target/deps/mathlib') + '/'
 
+bib = pybtex.database.parse_file(f'{local_lean_root}docs/references.bib')
+
+label_style = LabelStyle()
+# cf. LabelStyle.format_labels in pybtex.style.labels.alpha:
+# first generate label from author(s) + year
+# counted will be the total number of times the label appears
+counted = Counter()
+
+latexnodes2text = LatexNodes2Text()
+def clean_tex(src: str) -> str:
+  return latexnodes2text.latex_to_text(src)
+
+for key, data in bib.entries.items():
+  for author in data.persons['author']:
+    # turn LaTeX special characters to Unicode,
+    # since format_label does not correctly abbreviate names containing LaTeX
+    author.last_names = list(map(clean_tex, author.last_names))
+  label = label_style.format_label(data)
+  counted.update([label])
+  data.fields['alpha_label'] = label
+# count will track number of times label has been finalized
+count = Counter()
+
+for key, data in bib.entries.items():
```

Comment:
The ordering of the bib entries here is whatever it is in mathlib's `docs/references.bib`. This could potentially lead to strange ordering of the a, b, c that are appended to labels.